cli/cmd/resources.go

```diff
@@ -35,16 +35,20 @@ var (
 var (
 	httpClient = &resourcemanager.HTTPClient{}
 
-	environmentClient = resourcemanager.NewClient(
+	variableSetPreprocessor = preprocessor.VariableSet(cliLogger)
+	variableSetClient       = resourcemanager.NewClient(
 		httpClient, cliLogger,
-		"environment", "environments",
+		"variableset", "variablesets",
 		resourcemanager.WithTableConfig(resourcemanager.TableConfig{
 			Cells: []resourcemanager.TableCellConfig{
 				{Header: "ID", Path: "spec.id"},
 				{Header: "NAME", Path: "spec.name"},
 				{Header: "DESCRIPTION", Path: "spec.description"},
 			},
 		}),
+		resourcemanager.WithResourceType("VariableSet"),
+		resourcemanager.WithApplyPreProcessor(variableSetPreprocessor.Preprocess),
```

Comment:
Preprocessor to switch `Environment` to `VariableSet` and DeprecatedAlias to pass the `resourceType` validations